core/db/migrate/20161014221052_add_available_to_columns_and_remove_display_on_from_payment_methods.rb

```diff
@@ -0,0 +1,28 @@
+class AddAvailableToColumnsAndRemoveDisplayOnFromPaymentMethods < ActiveRecord::Migration[5.0]
+  def up
+    add_column(:spree_payment_methods, :available_to_users, :boolean, default: nil)
```

Comment:
I agree. Originally it was meant to be `default: true` and avoid tri-value logic, however the edge case of `nil` valued `display_on`s make tri-value logic necessary.